pub/assets/js/hound.js

```diff
@@ -284,12 +284,13 @@ var Model = {
   UrlToRepo: function(repo, path, line) {
     var info = this.repos[repo],
         url = info.url.replace(/\.git$/, ''),
-        anc = line ? '#L' + line : '';
+        anc = line ? info.anchorpattern.replace('${line}', line) : '';
 
     // Hacky solution to fix _some_ of the 404's when using SSH style URLs
     url = url.replace("git@github.com:", 'https://www.github.com/');
```

Comment:
With this way this should be possible already. The Stash URLs for the browser are different than the VCS URLs as well:

VCS (git remote): https://stash.tld/scm/<STASH-PROJECT>/<REPO-NAME>.git
URL in Browser: https://stash.faz.net/projects/<STASH-PROJECT>/repos/<REPO-NAME>/browse/${path}${anc}

So setting urlpattern to https://www.github.com/etsy/Hound//blob/master/${path}${anc} should work.